test/testglobal.js

```diff
@@ -66,7 +66,7 @@ export const testglobal = {
     
     testGetVersion: function(test) {
         test.expect(1);
-        test.equal(ilib.getVersion().substring(0,5), "14.18");
+        test.equal(ilib.getVersion().substring(0,5), "14.19");
```

Comment:
I added new API (getCLDRVersion) in v14.19.
Doesn't need to be added here?

Reply:
The point of these tests is to verify that the es6 wrappers around the older ilib code work properly. They are not extensive and exhaustive tests of the functionality of ilib itself. We could add another test here, but I think the point is proven that (a) the wrapper works as intended and (b) we have the expected version of ilib.

Reply:
Oh wait, you mean, we need to add the wrapper  for getCLDRVersion, not a test for it?